src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          <label for="Surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            id="Surname"
+          >
+        </div>
+
+        <div class="form_field">
+          <label for="Name">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+            id="Name"
+          >
+        </div>
+
+        <div class="form_field">
+          <label for="HowOldAreYou">How old are You?</label>
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+            id="HowOldAreYou"
+          >
+        </div>
+
+        <div class="form_field">
+          <label for="FullDateOfBirth">Full date of birth:</label>
+          <input
+            type="date"
+            name="DateOfBirth"
```

Comment:
```suggestion
            name="dateOfBirth"
```
do you now what camelCase is? 
you have example in checklist
![Знімок екрана 2022-09-28 о 12 44 32](https://user-images.githubusercontent.com/106310796/192747098-c5533360-57f0-42a2-8fbc-81d707f68771.png)